packages/indexer-cli/src/commands/indexer/rules/get.ts

```diff
@@ -51,10 +56,18 @@ module.exports = {
       // Create indexer API client
       const client = await createIndexerManagementClient({ url: config.api })
 
-      const ruleOrRules =
-        identifier === 'all'
-          ? await indexingRules(client, !!merged)
-          : await indexingRule(client, identifier, !!merged)
+      let ruleOrRules
+      if (identifier === 'all') {
+        ruleOrRules = await indexingRules(client, !!merged, protocolNetwork)
+      } else {
+        if (!protocolNetwork) {
```

Comment:
Since the protocol network is now part of the `IndexingRuleIdentifier` type, we need it to identify (and lookup) a single indexing rule uniquely.

Reply:
Does it really make sense to require protocol network whenever indexer-cli is used to get an indexing rule? Then a user might need to always specify "goerli" even though they only indexes "goerli" subgraph right? 
maybe it should only be required if the result isn't unique already or just print out the results that matches identifier if protocol network isn't specified?

Reply:
It would also be nice if a user could set a default protocolNetwork value during a indexer-cli worksession and from then on it would just use that value for all commands so you don't have to type it on everything?

Reply:
I think it's possible. Could the `indexer connect` config file hold that information? If so, would that be a good place to store the default value?